hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -795,6 +796,17 @@ solExpr sp = \case
     return $ parens $ c <+> "?" <+> (mo' <> "._" <> pretty vn) <+> ":" <+> da'
   DLE_GetUntrackedFunds {} -> impossible "getUntrackedFunds"
   DLE_ContractNew {} -> impossible "contractNew"
+  DLE_ContractFromAddress at addr -> do
```

Comment:
You should make this a call to a function in the sol stdlib. Then make Maybe Address a built in type so it can be returned directly OR catch this call at the DLLet level and make it so you can assign the fields directly